AdobeStockImageAdminUi/Model/FilterParametersProvider/LicenseData.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImageAdminUi\Model\FilterParametersProvider;
+
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Framework\DB\Select;
+use Magento\MediaGalleryUi\Model\SelectModifierInterface;
+
+/**
+ * Apply license data
+ */
+class LicenseData implements SelectModifierInterface
```

Comment:
This class name does not eliminate correctly what data it provides. There is no license data. Only license state: is or not. I can suggest renaming this class and related functionality to `IsLicensed.`